.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,10 @@
+# Run commitlint on the commit messages in a pull request.
+
+name: Lint Commit Messages
+
+on:
+  - pull_request
+
+jobs:
+  commitlint:
+    uses: openedx/.github/.github/workflows/commitlint.yml@master
```

Comment:
@feanil, @e0d, looks like the addition of the `commitlint.yml` is now making CI fail.  Not knowing much about Backstage, I don't know whether to recomend ditching the check for this repo, or if it's actually applicable here, to fix it so it works.